draft-ietf-quic-transport.md

```diff
@@ -851,11 +852,10 @@ limits are set in the transport parameters (see
 using MAX_STREAMS frames ({{frame-max-streams}}). Separate limits apply to
 unidirectional and bidirectional streams.
 
-If a max_streams transport parameter or MAX_STREAMS frame is received with a
-value greater than 2^60, this allows a maximum stream ID that cannot be
-expressed as a variable-length integer (see {{integer-encoding}}).
-If either is received, the connection MUST be immediately closed with a
-connection error of type STREAM_LIMIT_ERROR (see {{immediate-close}}).
+A max_streams transport parameter or MAX_STREAMS frame is invalid if it has a
+value greater than 2^60, which would allow for an invalid maximum stream ID.
```

Comment:
It's not a maximum stream ID, it's just a stream ID that's larger than the maximum.